gnd/src/main/res/values/strings.xml

```diff
@@ -138,6 +138,7 @@
   <string name="sign_out">Sign out</string>
   <string name="cancel">Cancel</string>
   <string name="join_survey">Join survey</string>
+  <string name="collect_data">Collect data</string>
```

Comment:
Consider renaming to something that makes it more obvious this is a temporary hook (eg "Test new collect data UI")

Reply:
Sgtm.